cmd/ical-relay/templates/calendar.html

```diff
@@ -141,7 +141,7 @@
                     currentType = "month";
                 }
             }
-            currentDate = location.hash ? dayjs(location.hash.substring(1)) : dayjs().startOf(currentType);
+            currentDate = location.hash ? dayjs(location.hash.substring(1)) : dayjs().startOf('day');
```

Comment:
This change should be removed when the check in updateCalendar is moved to setSelectedDate.